docs/howto/manage-charms.rst

```diff
@@ -65,6 +66,34 @@ in, a charm name different from the name of the root directory, etc.
 
     See more: :ref:`manage-extensions`
 
+The profiles also all provide recommended tooling and configuration for
+developing charms:
+
+- Run ``tox -e format`` to automatically format the charm and test code.
+- Run ``tox -e lint`` to lint the charm.
+- Run ``tox -e unit`` to run the charm unit tests.
+- Run ``tox -e static`` to run static type checking.
+- Run ``tox`` to run all of the above.
+- Run ``tox -e integration`` to run the charm integration tests.
+
+.. admonition:: Best Practice
+    :class: hint
+
+    The quality assurance pipeline of a charm should be automated using a
+    continuous integration (CI) system.
+
+.. tip::
+
+    To prepare an environment for running integration tests, such as in CI, use
+    the `concierge tool <https://github.com/jnsgruk/concierge>`_ or the
+    `actions-operator <https://github.com/charmed-kubernetes/actions-operator>`_.
+
+.. tip::
+
+    The `charming-actions <https://github.com/canonical/charming-actions>`_
+    repository includes actions to ensure that libraries are up-to-date, publish
+    charms and libraries, and more.
+
```

Comment:
@lengau Would you say these two things we have here as tips are in fact best practices too?